platform/windows/display_server_windows.cpp

```diff
@@ -2442,8 +2442,10 @@ LRESULT DisplayServerWindows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARA
 
 					if (motion > 0) {
 						mb->set_button_index(MOUSE_BUTTON_WHEEL_UP);
+						mb->set_factor(fabs((double)motion / (double)WHEEL_DELTA));
 					} else {
 						mb->set_button_index(MOUSE_BUTTON_WHEEL_DOWN);
+						mb->set_factor(fabs((double)motion / (double)WHEEL_DELTA));
```

Comment:
If it's the same call for both branches, it can go outside the `if`.

Reply:
Right, I cleaned up the code for the horizontal mousewheel event as well now. Do I need to squash the commits?

Reply:
Yes, please do.